modules/rhcos-enabling-multipath.adoc

```diff
@@ -0,0 +1,134 @@
+----
+
+. Check the machine configs to see that the new one was added:
++
+[source,terminal]
+----
+$ oc get MachineConfig
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
+00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+99-master-kargs-mpath
+52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
+99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+99-worker-ssh                              3.2.0             40m
+52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+----
+
+. Check the nodes:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                           STATUS                     ROLES    AGE   VERSION
+ip-10-0-136-161.ec2.internal   Ready                      worker   28m   v1.20.0
+ip-10-0-136-243.ec2.internal   Ready                      master   34m   v1.20.0
+ip-10-0-141-105.ec2.internal   Ready,SchedulingDisabled   worker   28m   v1.20.0
+ip-10-0-142-249.ec2.internal   Ready                      master   34m   v1.20.0
+ip-10-0-153-11.ec2.internal    Ready                      worker   28m   v1.20.0
+ip-10-0-153-150.ec2.internal   Ready                      master   34m   v1.20.0
+----
++
+You can see that scheduling on each worker node is disabled as the change is being applied.
+
+. Check that the kernel argument worked by going to one of the worker nodes and listing
+the kernel command line arguments (in `/proc/cmdline` on the host):
++
+[source,terminal]
+----
+$ oc debug node/ip-10-0-141-105.ec2.internal
+----
++
+.Example output
+[source,terminal]
+----
+Starting pod/ip-10-0-141-105ec2internal-debug ...
+To use host binaries, run `chroot /host`
+
+sh-4.2# cat /host/proc/cmdline
+BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+rootflags=defaults,prjquota rw root=UUID=fd0... ostree=/ostree/boot.0/rhcos/16...
+coreos.oem.id=qemu coreos.oem.id=ec2 ignition.platform.id=ec2 mpath=0
```

Comment:
Those kargs look like they're from Container Linux? The kargs expected are essentially the same ones added in the MC.

Reply:
Ah, I see. I had copied over the MC validation steps from another procedure and couldn't test, so didn't realize that's what the expected output should be here. 🤦 
Does this look right now, @jlebon?